### DISCUSSION TITLE
Disconnecting & Reconnecting monitor support

### DISCUSSION THREAD

**Discussion by jbru-bliq** [kangyu-california/PersistentWindows#244] [Q&A] [answered]
Can this tool handle maintaining the position of windows when a monitor is disconnected and reconnected? 

For context, I have two monitors that may be disconnected at any point. I am running Firefox on the bottom and Chrome on the top. 

1. If the top monitor is disconnected, the bottom should continue to show Chrome
2. If the bottom monitor is disconnected, Chrome should be moved to the top (preferred) or Firefox should be displayed
3. When monitors are reconnected (in any permutation), Chrome should be on the bottom and Firefox should be on the top

Is this within PersistentWindows toolset?

**Comment by kangyu-california** [comment, marked as answer]
Sounds like a typical scenario PW is designed to cover.

**Reply by jbru-bliq** [reply]
Thank you, this worked perfectly.
